packages/medusa/src/types/product-category.ts

```diff
@@ -15,6 +17,7 @@ export type UpdateProductCategoryInput = {
   is_internal?: boolean
   is_active?: boolean
   parent_category_id?: string | null
```

Comment:
question: should we add `transformParentIdToEntity` in the `update` method as well?

Reply:
Its already there. Its just hidden in the git lines.

Reply:
ahh indeed 😅